### PR TITLE
Update for Biblio related features

### PR DESCRIPTION
**Pull request by DoItTomorrow**
Added BiblioEdit command and supporting functionalities
Biblio command now generates usable biblio entries.

#75 #123

To do: Add documentation for relevant functionalities

### CODE REVIEW

**Review by suriruhani — APPROVED** [review]

File: src/main/java/seedu/address/logic/parser/SourceManagerParser.java [hunk 1]
```diff
@@ -188,6 +190,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case BiblioCommand.COMMAND_WORD:
             return new BiblioCommandParser().parse(arguments);
 
+        case BiblioEditCommand.COMMAND_WORD:
+            return new BiblioEditCommandParser().parse(arguments);
+
+        case RestoreCommand.COMMAND_WORD:
```
Comment (case141) [inline src/main/java/seedu/address/logic/parser/SourceManagerParser.java:196]: why did you add restorecommand back in sourcemanagerparser here?
